src/pages/Transaction/TransactionComp/TransactionComp.tsx

```diff
@@ -18,18 +23,57 @@ const handleCellbaseInputs = (inputs: Cell[], outputs: Cell[]) => {
 }
 
 export const TransactionComp = ({ transaction }: { transaction: Transaction }) => {
```

Comment:
Based on the requirements here: https://github.com/Magickbase/ckb-explorer-frontend/pull/177#issuecomment-1865818287

This component should not be modified to use pagination mode, as it would result in multiple pages making a large number of requests, as seen in #177.

Reply:
this file is meant to be changed, I reverted `src/components/TransactionItem/index.tsx`